src/main/kotlin/uk/gov/justice/probation/courtcaseservice/controller/CaseWorkflowController.kt

```diff
@@ -33,7 +33,7 @@ class CaseWorkflowController(val caseWorkflowService: CaseWorkflowService, val a
     @Operation(description = "Fetch hearing outcomes")
     @GetMapping(value = ["/courts/{courtCode}/hearing-outcomes"], produces = [APPLICATION_JSON_VALUE])
     fun fetchHearingOutcomes(@PathVariable("courtCode") courtCode: String, @Valid hearingOutcomeSearchRequest: HearingOutcomeSearchRequest): HearingOutcomeCaseList {
-        val cases = caseWorkflowService.fetchHearingOutcomes(courtCode, hearingOutcomeSearchRequest)
-        return HearingOutcomeCaseList(cases)
+        val cases = caseWorkflowService.getOutcomeCountsByState(courtCode, hearingOutcomeSearchRequest)
```

Comment:
The name is misleading. getOutcomeByState() but the state is not part of it! what is wrong with the previous name?

Reply:
Sorry, that was not supposed to be renamed. Let me fix

Reply:
fixed